packages/website/components/account/storageManager/storageManager.js

```diff
@@ -85,6 +86,11 @@ const StorageManager = ({ className = '', content }) => {
 
   return (
     <div ref={storageManagerRef} className={clsx('section storage-manager-container', className)}>
+      <Link href={'account/payment'} passHref>
+        <a href={'account/payment'} className="storage-manager-payment-link">
+          {currentPlan?.id !== 'pro' ? 'Want more storage? Upgrade your plan here!' : 'Plans & Payment'}
```

Comment:
Why is this the logic? We only want to show it if you're not on pro? Why not show it if you're on 'free' or 'null'? If it's not obvious, maybe explain with a comment or good variable name for the boolean? e.g. if we change the tier names or number of tiers later, how will we know to remember to change this or whether or not it makes sense to?

Reply:
If pro is the highest plan there is nothing to upgrade. It's honestly trivial we can remove it and just keep the same link text since pricing is in flux.